go/client/cmd_wallet_handle_uri.go

```diff
@@ -0,0 +1,301 @@
+		ui.Printf("Note: Keybase will NOT be submitting the transaction to the Stellar\n")
+		ui.Printf("network and does not know what will happen to it after sending\n")
+		ui.Printf("it to the URL above.\n")
+
+		if err := ui.PromptForConfirmation(fmt.Sprintf("Sign transaction to pay %s %s to %s and send it to the URL above?", c.green(v.Amount), c.green(assetDisplay), c.yellow(v.Recipient), v.CallbackURL)); err != nil {
+			return err
+		}
+	}
+
+	// user approved the operation, so proceed
+	cli, err := GetWalletClient(c.G())
+	if err != nil {
+		return err
+	}
+	protocols := []rpc.Protocol{
+		NewIdentifyUIProtocol(c.G()),
+	}
+	if err := RegisterProtocolsWithContext(protocols, c.G()); err != nil {
+		return err
+	}
+
+	if v.AssetCode == "" {
+		arg := stellar1.ApprovePayURILocalArg{
+			InputURI: c.uri,
+			Amount:   v.Amount,
+			FromCLI:  true,
+		}
+		txID, err := cli.ApprovePayURILocal(context.Background(), arg)
+		if err != nil {
+			return err
+		}
+
+		if v.CallbackURL == "" {
+			ui.Printf("Signed transaction sent to %s\n", v.CallbackURL)
+		}
+		ui.Printf("Sent!\nStellar Transaction ID: %v\n", txID)
+	} else {
+		// find a path payment so user can confirm it is ok
+		arg := stellar1.FindPaymentPathLocalArg{
+			To:     v.Recipient,
+			Amount: v.Amount,
+			DestinationAsset: stellar1.Asset{
+				Code:   v.AssetCode,
+				Issuer: v.AssetIssuer,
+			},
+		}
+		ui.Printf(c.yellow(fmt.Sprintf("Searching for payment path to %s...", arg.DestinationAsset)) + "\n")
+		path, err := cli.FindPaymentPathLocal(context.Background(), arg)
+		if err != nil {
+			return err
+		}
+
+		ui.Printf("Sending approximately %s of %s (at most %s)\n", path.FullPath.SourceAmount, path.FullPath.SourceAsset, path.FullPath.SourceAmountMax)
+		ui.Printf("Account %s will receive %s of %s\n\n", arg.To, path.FullPath.DestinationAmount, path.FullPath.DestinationAsset)
+
+		if err := ui.PromptForConfirmation("Proceed?"); err != nil {
+			return err
+		}
+
+		approveArg := stellar1.ApprovePathURILocalArg{
+			InputURI: c.uri,
+			FullPath: path.FullPath,
+			FromCLI:  true,
+		}
+		txID, err := cli.ApprovePathURILocal(context.Background(), approveArg)
+		if err != nil {
+			return err
+		}
+		if v.CallbackURL == "" {
+			ui.Printf("Signed transaction sent to %s\n", v.CallbackURL)
+		}
+		ui.Printf("Sent!\nStellar Transaction ID: %v\n", txID)
+	}
+
+	return nil
+}
+
+func (c *CmdWalletHandleURI) txOp(v stellar1.ValidateStellarURIResultLocal) error {
+	ui := c.G().UI.GetTerminalUI()
+	ui.Printf("The URI is requesting that you sign a transaction.\n\n")
+	ui.Printf("Here is the base64-encoded transaction envelope:\n\n")
+	ui.Printf("\t%s\n\n", v.Xdr)
+	ui.Printf("Transaction summary:\n\n")
+	if v.Summary.Source != "" {
+		ui.Printf("\tSource account: %s\n", v.Summary.Source)
+	}
+	if v.Summary.Fee > 0 {
+		ui.Printf("\tFee: %d stroops\n", v.Summary.Fee)
+	}
+	if v.Summary.Memo != "" {
+		ui.Printf("\tMemo: %q (%s)\n", v.Summary.Memo, v.Summary.MemoType)
```

Comment:
memo question again

Reply:
This CLI command is temporary because we don't have frontend support yet.

Reply:
can it be unlisted then if it might have that problem?